fedmsg_meta_fedora_infrastructure/pagure.py

```diff
@@ -231,13 +231,13 @@ def subtitle(self, msg, **config):
                 user=user, project=project, id=issueid, tags=tags)
         elif 'pagure.issue.assigned.added' in msg['topic']:
             issueid = msg['msg']['issue']['id']
-            if msg['msg']['issue']['assignee'] is None:
-                # https://pagure.io/pagure/issue/1896
-                assignee = '?? Someone ??'
-            else:
-                assignee = msg['msg']['issue']['assignee']['name']
+            assignee = ''
+            if msg['msg']['issue']['assignee']:
+                assignee = ' to {0}'.format(
+                    msg['msg']['issue']['assignee']['name']
+                )
             tmpl = self._(
-                '{user} assigned ticket {project}#{id} to {assignee}')
+                '{user} assigned ticket {project}#{id}{assignee}')
```

Comment:
Sorry I missed responding to this, I realize my comment wasn't very clear. I think it's better to not try to compress it into a one-liner at the expense of readability. However, you should do what you think is clearest since "don't really like" isn't a strong argument one way or the other.